vrf/vrf_pbc.go

```diff
@@ -0,0 +1 @@
+package vrf
```

Comment:
this go file is used for?

Reply:
will implement the vrf scheme for curve with pairing, such as bn128.